sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/EncryptParameters.java

```diff
@@ -0,0 +1,322 @@
+     * {@link EncryptionAlgorithm#A256CBCPAD}.
+     *
+     * @param plainText The content to be encryption.
+     *
+     * @return The {@link EncryptParameters}.
+     */
+    public static EncryptParameters createAes256CbcPadParameters(byte[] plainText) {
+        return createAes256CbcPadParameters(plainText, null);
+    }
+
+    /**
+     * Factory method to create an instance of {@link EncryptParameters} with the given parameters for
+     * {@link EncryptionAlgorithm#A256CBCPAD}.
+     *
+     * @param plainText The content to be encryption.
+     * @param iv Initialization vector for the encryption operation.
+     *
+     * @return The {@link EncryptParameters}.
+     */
+    public static EncryptParameters createAes256CbcPadParameters(byte[] plainText, byte[] iv) {
+        return new EncryptParameters(EncryptionAlgorithm.A256CBCPAD, plainText, iv, null);
+    }
+
+    /**
+     * Factory method to create an instance of {@link EncryptParameters} with the given parameters for
+     * {@link EncryptionAlgorithm#A256GCM}.
+     *
+     * @param plainText The content to be encryption.
+     * @param iv Initialization vector for the encryption operation.
+     *
+     * @return The {@link EncryptParameters}.
+     */
+    public static EncryptParameters createAes256GcmParameters(byte[] plainText, byte[] iv) {
+        return createAes256GcmParameters(plainText, iv, null);
+    }
+
+    /**
+     * Factory method to create an instance of {@link EncryptParameters} with the given parameters for
+     * {@link EncryptionAlgorithm#A256GCM}.
+     *
+     * @param plainText The content to be encryption.
+     * @param iv Initialization vector for the encryption operation.
+     * @param additionalAuthenticatedData Additional data to authenticate when using authenticated crypto algorithms.
+     *
+     * @return The {@link EncryptParameters}.
+     */
+    public static EncryptParameters createAes256GcmParameters(byte[] plainText, byte[] iv,
+                                                              byte[] additionalAuthenticatedData) {
+        return new EncryptParameters(EncryptionAlgorithm.A256GCM, plainText, iv, additionalAuthenticatedData);
+    }
+
+    /**
+     * Creates an instance of {@link EncryptParameters} with the given parameters.
+     *
+     * @param algorithm The algorithm to be used for encryption.
+     * @param plainText The content to be encrypted.
+     * @param iv Initialization vector for the encryption operation.
+     * @param additionalAuthenticatedData Additional data to authenticate when using authenticated crypto algorithms.
+     */
+    EncryptParameters(EncryptionAlgorithm algorithm, byte[] plainText, byte[] iv, byte[] additionalAuthenticatedData) {
+        Objects.requireNonNull(algorithm, "Encryption algorithm cannot be null.");
+        Objects.requireNonNull(plainText, "Plain text content to be encrypted cannot be null.");
```

Comment:
Just "Plaintext". I may have missed this for "Ciphertext" above as well.

Reply:
Please refer to [my comment](https://github.com/Azure/azure-sdk-for-java/pull/20357#discussion_r607426548) above.